src/Npgsql/Npgsql.csproj

```diff
@@ -133,9 +133,11 @@
     <Reference Include="System.Core" />
     <Reference Include="System.Data" />
     <Reference Include="System.DirectoryServices" />
+    <Reference Include="System.Drawing" />
     <Reference Include="System.Numerics" />
     <Reference Include="System.Transactions" />
     <Reference Include="System.Configuration" />
+    <Reference Include="System.Windows.Forms" />
```

Comment:
Should there actually be a dependency on System.Windows.Forms and on System.Drawing?